src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -969,6 +1482,13 @@ CacheEntry fileCacheGetter( const std::pair< std::string, ImageReader::ChannelIn
 
 	const std::string &fileName = fileNameAndChannelInterpretation.first;
 
+	if( !g_disableExrCore )
+	{
+		OIIO::attribute( "openexr:core", 1 );
+	}
+
+	OIIO::attribute( "threads", 1 );
```

Comment:
> Should the number of threads be 0 rather than 1?

No, I don't know if this is especially well documented, but looking at the code: https://github.com/AcademySoftwareFoundation/OpenImageIO/blob/372bb8eca379013cc43e2f39b395dcbf4f90e5b7/src/libOpenImageIO/imageio.cpp#L381

... you can see that 0 means "use the default", and any for any other number it subtracts 1 to get the number of worker threads ... I guess this is because 1 is supposed to be the number of total threads, so 1 means "just the main thread, no worker threads".